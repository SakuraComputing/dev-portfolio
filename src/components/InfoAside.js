import React from 'react';

export const InfoAside = (props) => (
    <aside className="info aside section">
        <div className="section-inner">
            <h2 className="heading sr-only">Basic Information</h2>
            <div className="content">
                <ul className="list-unstyled">
                    <li><i className="fa fa-map-marker"></i><span className="sr-only">Location:</span>{props.infoaside.location}
                    </li>
                    <li><i className="fa fa-envelope-o"></i><span className="sr-only">Email:</span><a
                        href="#">{props.infoaside.email}</a></li>
                    {/* <li><i className="fa fa-link"></i><span className="sr-only">Website:</span><a
                        href="#">{props.infoaside.website}</a></li> */}
                </ul>
            </div>
        </div>
    </aside>
);

export default InfoAside;