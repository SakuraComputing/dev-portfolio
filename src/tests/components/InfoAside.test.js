import React from 'react';
import { shallow } from 'enzyme';
import InfoAside from '../../components/InfoAside';

describe('InfoAside Section', () => {
    it('should render correctly', () => {
        const infoaside = {
          location: "Manc",
          email: "a@e.gmail.com",
          website: "www.test.com"
        };
        const wrapper = shallow(<InfoAside infoaside={infoaside} />);
        expect(wrapper).toMatchSnapshot();
    });
});