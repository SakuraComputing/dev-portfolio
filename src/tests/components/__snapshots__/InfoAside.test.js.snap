// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoAside Section should render correctly 1`] = `
<aside
  className="info aside section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading sr-only"
    >
      Basic Information
    </h2>
    <div
      className="content"
    >
      <ul
        className="list-unstyled"
      >
        <li>
          <i
            className="fa fa-map-marker"
          />
          <span
            className="sr-only"
          >
            Location:
          </span>
          Manc
        </li>
        <li>
          <i
            className="fa fa-envelope-o"
          />
          <span
            className="sr-only"
          >
            Email:
          </span>
          <a
            href="#"
          >
            a@e.gmail.com
          </a>
        </li>
      </ul>
    </div>
  </div>
</aside>
`;
